bin/audit.py

```diff
@@ -0,0 +1,152 @@
+# from gfactoryXmlToDB import *
+from collections import defaultdict
+import yaml
+import requests
```

Comment:
It doesn't appear as if you use any of these libraries, though you'll want to keep `yaml`. Additionally, `requests` duplicates the functionality of `urllib.requests`

A linter should warn you for these sorts of things. I suggest https://pylint.org/.

Reply:
Yeah I had the linter saying they were unused and I focused on the bottom parts. I'll figure out the yaml part as I still may need to parse one yml file...?